std/utf.d

```diff
@@ -3110,7 +3110,8 @@ enum dchar replacementDchar = '\uFFFD';
  * Params:
  *      r = input range of characters, or array of characters
  * Returns:
- *      input range
+ *      An input range with a length if `isAggregateType!R` is `true`.
+ *      Otherwise, this returns a random access range with slicing.
```

Comment:
Actually, it does: https://github.com/dlang/phobos/blob/master/std/range/primitives.d#L912